packages/editor/src/components/rich-text/index.js

```diff
@@ -612,7 +573,7 @@ export class RichText extends Component {
 		// The input event does not fire when the whole field is selected and
 		// BACKSPACE is pressed.
 		if ( keyCode === BACKSPACE ) {
-			this.onChange( this.createRecord(), true );
```

Comment:
I don't understand why this change has been made.

Reply:
We used to skip applying changes back to the live DOM, and so did we here. Since #11595 we no longer do that, but it hasn't been updated here. I've removed it here while revising the parameters for `this.onChange`.